src/monorepo-shipit/src/RepoGit.js

```diff
@@ -110,13 +111,7 @@ export default class RepoGit implements AnyRepo, SourceRepo, DestinationRepo {
   };
 
   findLastSourceCommit: (roots: Set<string>) => null | string = (roots) => {
-    const log = this._gitCommand(
-      'log',
-      '-1',
-      '--grep',
-      '^adeira-source-id: \\?[a-z0-9]\\+\\s*$',
-      ...roots,
-    )
+    const log = this._gitCommand('log', '-1', '--grep', '^adeira-source-id: [a-z0-9]*$', ...roots)
```

Comment:
This causes one unit test to fail. Oddly enough if I revert it and run the tests locally they all fail. Is this a problem with bash vs. zsh. vs. ?

Reply:
For now I've reverted with the hope that it works fine in CI just not during my local. I'm going to make a git patch to find out.

Reply:
Yes that seems to be the case. Odd!

<img width="291" alt="image" src="https://github.com/adeira/universe/assets/6801309/463db660-dd59-4c90-bda9-d7a9e6613ce0">

Reply:
@itsdouges I also had the failing test locally, but not anymore. I described the issue here: https://github.com/adeira/universe/pull/6114 (second half of the description)

TL;DR: `brew install git` because you _probably_ have the older broken version of Git on macOS.

Reply:
Thank you!